account-kit/react/src/icons/EOAConnectionFailed.tsx

```diff
@@ -1,140 +1,207 @@
-            values="0 0 0 0 0.901961 0 0 0 0 0.466667 0 0 0 0 0.176471 0 0 0 0.12 0"
-          />
-          <feBlend
-            mode="normal"
-            in2="effect1_dropShadow_49_9053"
-            result="effect2_dropShadow_49_9053"
-          />
-          <feColorMatrix
-            in="SourceAlpha"
-            type="matrix"
-            values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
-            result="hardAlpha"
-          />
-          <feOffset dx="4" dy="36" />
-          <feGaussianBlur stdDeviation="18" />
-          <feColorMatrix
-            type="matrix"
-            values="0 0 0 0 0.901961 0 0 0 0 0.466667 0 0 0 0 0.176471 0 0 0 0.1 0"
-          />
-          <feBlend
-            mode="normal"
-            in2="effect2_dropShadow_49_9053"
-            result="effect3_dropShadow_49_9053"
-          />
-          <feColorMatrix
-            in="SourceAlpha"
-            type="matrix"
-            values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
-            result="hardAlpha"
-          />
-          <feOffset dx="8" dy="81" />
-          <feGaussianBlur stdDeviation="24.5" />
-          <feColorMatrix
-            type="matrix"
-            values="0 0 0 0 0.901961 0 0 0 0 0.466667 0 0 0 0 0.176471 0 0 0 0.06 0"
-          />
-          <feBlend
-            mode="normal"
-            in2="effect3_dropShadow_49_9053"
-            result="effect4_dropShadow_49_9053"
-          />
-          <feColorMatrix
-            in="SourceAlpha"
-            type="matrix"
-            values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
-            result="hardAlpha"
-          />
-          <feOffset dx="14" dy="143" />
-          <feGaussianBlur stdDeviation="29" />
-          <feColorMatrix
-            type="matrix"
-            values="0 0 0 0 0.901961 0 0 0 0 0.466667 0 0 0 0 0.176471 0 0 0 0.02 0"
-          />
-          <feBlend
-            mode="normal"
-            in2="effect4_dropShadow_49_9053"
-            result="effect5_dropShadow_49_9053"
-          />
-          <feBlend
-            mode="normal"
-            in="SourceGraphic"
-            in2="effect5_dropShadow_49_9053"
-            result="shape"
-          />
-        </filter>
-        <linearGradient
-          id="paint0_linear_49_9053"
-          x1="0"
-          y1="70"
-          x2="140"
-          y2="70"
-          gradientUnits="userSpaceOnUse"
-        >
-          <stop stopColor="#FAE0D1" />
-          <stop offset="1" stopColor="#FDF2CE" />
-        </linearGradient>
-        <clipPath id="clip0_49_9053">
-          <rect width="140" height="140" rx="70" fill="white" />
-        </clipPath>
-      </defs>
+      {/* Metamask logo */}
+      {walletType === EOAWallets.METAMASK && <MetamaskIconPath />}
+      {/* Coinbase logo */}
+      {walletType === EOAWallets.COINBASE_WALLET && <CoinbaseIconPath />}
+      {/* WalletConnect logo */}
+      {walletType === EOAWallets.WALLET_CONNECT && <WalletConnectIconPath />}
```

Comment:
this isn't super scalable. I think we need 3 other things instead:
1. for things like Coinbase and Metamask (injected wallets), the connectors provide an optional logoUrl. We should use that here
2. a fallback, generic icon in the case that an injected wallet doesn't have a logoUrl
3. for walletconnect what we have here is good